### ISSUE TITLE
Missing man page

### ISSUE THREAD

**Issue by fuzzy-id** [edolstra/nix-repl#14]
A man page describing what the program is all about, how it works and how to work with it would be nice to have.

**Comment by tsion** [comment]
@edolstra What do you think of moving nix-repl into https://github.com/NixOS/nix to make use of the build infrastructure there to add a manpage?

**Comment by edolstra** [comment]
Yeah, that's the plan, see https://gist.github.com/edolstra/efcadfd240c2d8906348. The `nix` command on the `new-cli` branch has autogenerated `--help` output BTW.
